From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Mon, 11 Oct 2021 17:09:14 -0500
Subject: [PATCH] Remove prebuilt jnm2.ReferenceAssemblies.net35

This prebuilt package filled a gap in the official 1.0 version of the .NET
Framework reference assembly packages. The gap has been fixed in later versions
of the official packages. Removal fixes a prebuilt dependency by letting the
repo use the official version of the packges, which are produced by SBRP.
https://github.com/dotnet/msbuild/issues/6935
---
 src/StringTools/StringTools.csproj | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/StringTools/StringTools.csproj b/src/StringTools/StringTools.csproj
index fef9909..93a4634 100644
--- a/src/StringTools/StringTools.csproj
+++ b/src/StringTools/StringTools.csproj
@@ -20,17 +20,12 @@
 
   <PropertyGroup Condition="'$(TargetFramework)' == 'net35'">
     <AssemblyName>Microsoft.NET.StringTools.net35</AssemblyName>
-    <!-- Disable Fx install checks as we're building against jnm2's 3.5 reference assemblies -->
-    <BypassFrameworkInstallChecks>true</BypassFrameworkInstallChecks>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' != 'net35'">
     <PackageReference Include="System.Memory" />
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" />
   </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="jnm2.ReferenceAssemblies.net35" PrivateAssets="All" />
-  </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net35'">
     <Compile Remove="SpanBasedStringBuilder.cs" />
